.github/workflows/release.yml

```diff
@@ -1,58 +1,72 @@
-name: Prerelease
+name: Release
 
 on:
   push:
     branches:
-      - 'master'
+      - master
 
 jobs:
-  publish-npm:
+  release:
     runs-on: ubuntu-latest
-    if: "!contains(github.event.head_commit.message, 'skip ci') && !contains(github.event.head_commit.message, 'chore(release): packages')"
+
     steps:
       - name: GitHub context
         run: echo "$GITHUB_CONTEXT"
         env:
           GITHUB_CONTEXT: ${{ toJson(github) }}
       - name: Checkout
-        if: success()
+        if: github.ref == 'refs/heads/master'
         uses: actions/checkout@v2
         with:
           ref: master
           fetch-depth: 50
-      - name: Fetch tags
-        if: success()
+
+      - name: Fetch all tags (for releases)
         run: git fetch --depth=1 origin +refs/tags/*:refs/tags/*
+
       - name: Read .nvmrc
-        if: success()
         run: echo ::set-output name=NVMRC::$(cat .nvmrc)
         id: nvm
-      - name: yarn cache directory
-        if: success()
+
+      - name: Setup Node (uses version in .nvmrc)
+        uses: actions/setup-node@v1
+        with:
+          node-version: "${{ steps.nvm.outputs.NVMRC }}"
+          registry-url: https://registry.npmjs.org/
+          scope: flopflip
+
+      - name: Get yarn cache
         id: yarn-cache
         run: echo "::set-output name=dir::$(yarn cache dir)"
-      - name: Setup node_modules cache
-        if: success()
-        uses: actions/cache@v1
+
+      - uses: actions/cache@v1
         with:
           path: ${{ steps.yarn-cache.outputs.dir }}
           key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
           restore-keys: |
             ${{ runner.os }}-yarn-
-      - name: Setup Node (uses version in .nvmrc)
-        if: success()
-        uses: actions/setup-node@v1
-        with:
-          node-version: '${{ steps.nvm.outputs.NVMRC }}'
-          registry-url: https://registry.npmjs.org/
-          scope: flopflip
-      - name: Install
-        if: success()
+      - name: Install dependencies
         run: yarn install --frozen-lockfile
-      - name: Publish to next dist-tag
-        if: success()
+
+      - name: Creating .npmrc
         run: |
-          echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc
-          yarn release:next
+          cat << EOF > "$HOME/.npmrc"
+            email=npmjs@commercetools.com
+            //registry.npmjs.org/:_authToken=$NPM_TOKEN
+          EOF
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Create Release Pull Request or Publish to npm
+        id: changesets
+        # uses: changesets/action@master
+        uses: emmenko/action@nm-fix-npmrc-overwrite
```

Comment:
PS: my PR got merged, you can use `changesets/action@master`

Reply:
Yeah. Thanks. We need to use that on app-kit too. The branch was deleted. Breaks stuff :).  Pushed to master.

Reply:
Ah right :)